README.md

```diff
@@ -176,4 +176,6 @@ browserWindow.LastName.Text ="Doe";
 
 // Click on the Save button
 browserWindow.Save.Click();
-```
\ No newline at end of file
+```
+
+Follow the steps from [this guide](http://www.codewrecks.com/blog/index.php/2011/06/06/witch-version-of-browser-is-used-by-the-webbrowser-control/) to use a more recent version of IE in the Object Recorder.
```

Comment:
I think we should summarize the steps required from that page instead of just providing a link. We should also indicate that this is for Win32 and/or WPF applications, as the article isn't very clear on which WebBrowser control is being referred to. I also think we should cite MSDN sources whenever possible. We may want to have a dedicated wiki page about this because it's more advanced and off-topic.